bin/commands/init/generate/index.ts

```diff
@@ -0,0 +1,102 @@
+/*
+// This program and the accompanying materials are made available
+// under the terms of the Eclipse Public License v2.0 which
+// accompanies this distribution, and is available at
+// https://www.eclipse.org/legal/epl-v20.html
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Copyright Contributors to the Zowe Project.
+*/
+
+import * as std from 'cm_std';
+import * as os from "cm_os";
+import * as xplatform from "xplatform";
+import * as fs from '../../../libs/fs';
+import * as config from '../../../libs/config';
+import * as common from '../../../libs/common';
+import * as stringlib from '../../../libs/string';
+import * as zosFs from '../../../libs/zos-fs';
+import * as zosJes from '../../../libs/zos-jes';
+
+export function execute(dryRun?: boolean) {
+  common.requireZoweYaml();
+  const ZOWE_CONFIG=config.getZoweConfig();
+
+  const prefix=ZOWE_CONFIG.zowe.setup?.dataset?.prefix;
+  if (!prefix) {
+    common.printErrorAndExit(`Error ZWEL0157E: Zowe dataset prefix (zowe.setup.dataset.prefix) is not defined in Zowe YAML configuration file.`, undefined, 157);
+  }
+
+  const runtimeDirectory=ZOWE_CONFIG.zowe.runtimeDirectory;
+  if (!runtimeDirectory) {
+    common.printErrorAndExit(`Error ZWEL0157E: Zowe runtime directory (zowe.runtimeDirectory) is not defined in Zowe YAML configuration file.`, undefined, 157);
+  }
+  
+  const tempFile = fs.createTmpFile();
+  zosFs.copyMvsToUss(ZOWE_CONFIG.zowe.setup.dataset.prefix + '.SZWESAMP(ZWEGENER)', tempFile);
+  let jclContents = xplatform.loadFileUTF8(tempFile, xplatform.AUTO_DETECT);
```

Comment:
My datasets were allocated as working and were deleted after couple days.
Anyway, I have started my tests and got [S0C4](https://github.com/zowe/zowe-common-c/issues/453) here.
It can be easily bypassed by checking return code from `copyMvsToUss`, I will try to fix the abend.